### PR TITLE
add isRunning() and openUrl(url)

### PR DESCRIPTION
**Pull request by Jonahss**
Depends on https://github.com/appium/node-simctl/pull/15

@imurchie for review?

### CODE REVIEW

File: test/simulator-e2e-specs.js [hunk 1]
```diff
@@ -145,6 +145,21 @@ function runTests (deviceType) {
       stat = await sim.stat();
       stat.state.should.equal('Shutdown');
     });
+
+    it.only('should detect if a sim is running', async function () {
```
Comment (imurchie) [inline test/simulator-e2e-specs.js]: Probably don't want this `only`

File: README.md [hunk 2]
```diff
@@ -132,10 +132,18 @@ Retrieve state information about the simulator. Returns an object:
...
 
+`async isRunning ()`
+
+Returns `true` if the simulator is running, `false` otherwise.
+
 `async run ()`
 
 Starts the simulator without any Instruments involvement, or application running.
 
+`async getUrl (url)`
```
Comment (imurchie) [inline README.md]: Should this be `openUrl`? And it needs a comment about only being in Xcode 7+.

File: lib/simulator-xcode-7.js [hunk 3]
```diff
@@ -47,6 +47,14 @@ class SimulatorXcode7 extends SimulatorXcode6 {
       'iPhone 6s Plus (9.2)': 'iPhone 6s Plus (9.2) [',
     };
   }
+
+  async openUrl (url) {
```
Comment (imurchie) [inline lib/simulator-xcode-7.js]: If this is only in Xcode 7, can we have some sort of thing in the 6 class so we won't get `undefined is not a function` errors? Just throw an error that makes sense?

**Comment by jlipps** [comment]
:+1: to isaac's comments

**Comment by Jonahss** [comment]
updated with @imurchie's comments :)

**Comment by imurchie** [comment]
:+1:

**Pull request merged**